src/Microsoft.AspNetCore.Server.Kestrel/Http/SocketInput.cs

```diff
@@ -16,7 +16,7 @@ public class SocketInput : ICriticalNotifyCompletion
 
         private readonly MemoryPool2 _memory;
         private readonly IThreadPool _threadPool;
-        private readonly ManualResetEventSlim _manualResetEvent = new ManualResetEventSlim(false);
+        private readonly ManualResetEventSlim _manualResetEvent = new ManualResetEventSlim(false, 0);
```

Comment:
In CoreFX case, they seem to have removed `false` also... should it be the same here?